webapp/channels/src/components/admin_console/system_users/system_users.tsx

```diff
@@ -306,46 +306,34 @@ export class SystemUsers extends React.PureComponent<Props, State> {
-                                    value={this.props.searchTerm}
-                                    onChange={this.handleSearchFiltersChange}
-                                    onSearch={this.onSearch}
-                                />
-                                <SystemUsersFilterTeam
-                                    options={this.props.teams}
-                                    value={this.props.teamId}
-                                    onChange={this.handleSearchFiltersChange}
-                                    onFilter={this.onFilter}
-                                />
-                                <SystemUsersFilterRole
-                                    value={this.props.filter}
-                                    onChange={this.handleSearchFiltersChange}
-                                    onFilter={this.onFilter}
-                                />
-                            </div>
-                            <SystemUsersList
-                                loading={this.state.loading}
-                                search={this.doSearch}
-                                nextPage={this.nextPage}
-                                usersPerPage={USERS_PER_PAGE}
-                                total={this.props.totalUsers}
-                                teams={this.props.teams}
-                                teamId={this.props.teamId}
-                                filter={this.props.filter}
-                                term={this.props.searchTerm}
-                                onTermChange={this.handleTermChange}
-                                mfaEnabled={this.props.mfaEnabled}
-                                enableUserAccessTokens={this.props.enableUserAccessTokens}
-                                experimentalEnableAuthenticationTransfer={this.props.experimentalEnableAuthenticationTransfer}
+                    <div className='admin-console__container'>
+                        <div className='system-users__filter-row'>
+                            <SystemUsersSearch
+                                value={this.props.searchTerm}
+                                onChange={this.handleSearchFiltersChange}
+                                onSearch={this.onSearch}
+                            />
+                            <SystemUsersFilterTeam
+                                options={this.props.teams}
+                                value={this.props.teamId}
+                                onChange={this.handleSearchFiltersChange}
+                                onFilter={this.onFilter}
+                            />
+                            <SystemUsersFilterRole
+                                value={this.props.filter}
+                                onChange={this.handleSearchFiltersChange}
+                                onFilter={this.onFilter}
                             />
                         </div>
+                        <SystemUsersList/>
```

Comment:
I assume this will be hooked up eventually, are we waiting to do the filter component re-design in a follow-up PR?

Reply:
yes this is just more of a design review pr for the table. filters are specific to the implementation instead of being part of generic table.